app/src/renderer/vuex/modules/user.js

```diff
@@ -64,7 +64,13 @@ export default ({ commit, node }) => {
 
   const actions = {
     async signIn ({ commit, state }, seedWords) {
-      let privkey = dg.mnemonicToPrivKey(seedWords)
+      let privkey
+      try {
+        privkey = dg.mnemonicToPrivKey(seedWords)
+      } catch (e) {
+        commit('notifyError', {title: 'Generating private key failed', body: e.message})
```

Comment:
Maybe this should say "Deriving private key failed" instead of "generating", since this is for deriving a key generated during the fundraiser.